odiglet/pkg/kube/runtime_details/shared.go

```diff
@@ -65,32 +64,37 @@ func runtimeInspection(pods []corev1.Pod) ([]common.LanguageByContainer, error)
 				continue
 			}
 
-			detectionResults := inspectors.DetectLanguage(processes)
-			if len(detectionResults) == 0 {
-				log.Logger.V(0).Info("no supported language detected for container in pod", "processes", processes, "pod", pod.Name, "container", container.Name, "namespace", pod.Namespace)
-				continue
-			}
-
-			if len(detectionResults) > 1 {
-				log.Logger.V(0).Info("multiple languages detected for pod container processes, selecting first one", "processes", processes, "pod", pod.Name, "container", container.Name, "namespace", pod.Namespace)
-			}
-
-			resultsMap[container.Name] = common.LanguageByContainer{
-				ContainerName: container.Name,
-				Language:      detectionResults[0].Language,
+			for _, process := range processes {
+				lang, err := inspectors.DetectLanguage(process)
+				if err != nil {
+					log.Logger.V(0).Info("no supported language detected for container in pod", "process", process, "pod", pod.Name, "container", container.Name, "namespace", pod.Namespace)
```

Comment:
```suggestion
					log.Logger.V(0).Info("no supported language detected for process in a container in pod", "process", process, "pod", pod.Name, "container", container.Name, "namespace", pod.Namespace)
```

This message will now be printed per process where it used to be printed per container

Reply:
Updated to only take the first process into account, which should be the same behavior as we have today